ui/app/mainui/SendModalHandler.qml

```diff
@@ -0,0 +1,167 @@
+import QtQuick 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
+
+import AppLayouts.Wallet.stores 1.0 as WalletStores
+import AppLayouts.Wallet.popups.simpleSend 1.0
+
+import shared.popups.send 1.0
+import shared.stores.send 1.0
+
+import utils 1.0
+
+QtObject {
+    id: root
+
+    required property var popupParent
+    required property int loginType
+    required property TransactionStore transactionStore
+    required property WalletStores.CollectiblesStore walletCollectiblesStore
+
+    // for ens flows
+    required property string myPublicKey
+    required property string ensRegisteredAddress
+    // TODO: This should probably be a property and not a function. Needs changes on  backend side
+    property var getStatusTokenKey: function() {}
+
+    // for sticker flows
+    required property string stickersMarketAddress
+    required property string stickersNetworkId
+
+    // Feature flag for single network send until its feature complete
+    required property bool simpleSendEnabled
+
+    function openSend(params = {}) {
+        if (!!root._sendModalInstance) {
+            return
+        }
```

Comment:
I think it's really not needed (the whole `_sendModalInstance`). Double open of send modal would be a serious logical problem in the app. Hiding it silently there is not a solution imo.

Reply:
sure, I can remove it. Generally its not possible to click anywhere while a popup is currently launched, so lkely it wont happen.